### ISSUE TITLE
Snippets don’t get automatically translated

### ISSUE THREAD

**Issue by TheoChevalier** [wagtail/wagtail-localize#540]
I’ve noticed a regression, snippets stopped being automatically translated after a sync when there’s a change in the source page:

 <img width="845" alt="Capture d’écran 2022-03-10 à 16 59 45" src="https://user-images.githubusercontent.com/1294206/157888254-e4cca833-a451-4d4f-bfc3-f437bc2d9fbf.png">
(The UI says "Not translated" in front of the 3rd snippet)

### STR:

- Have a snippet set as TranslatableField
- Edit the source page and pick a new snippet 
- Sync translations
- Edit the French page

### Actual result:
- The new snippet is not translated

### Expected result:
- The new snippet is automatically translated

**Comment by zerolab** [comment]
@TheoChevalier is this with v1.1?

**Comment by TheoChevalier** [comment]
Correct, this was with 1.1rc1 and I’m reproducing with 1.1 final

**Comment by zerolab** [comment]
This slipped a bit under my radar. @TheoChevalier was "Test 4" a new snippet that did not have any translations?

**Comment by TheoChevalier** [comment]
> This slipped a bit under my radar. @TheoChevalier was "Test 4" a new snippet that did not have any translations?

Yup, Test 4 was created right before adding it to the English page

**Comment by zerolab** [comment]
I may be missing something, but doesn't that then work as expected? ie. it is a new, non-translated snippets, so it shows as such after you run "sync translations" which updates the translated pages with content from the source page.
